kvmagent/kvmagent/test/utils/my_debugger.py

```
@@ -0,0 +1,366 @@
+    import os
+    for path in sys.path:
+        zstacklib_path = os.path.join(path, "zstacklib")
+        if os.path.exists(zstacklib_path):
+            print("Found zstacklib directory at: " + zstacklib_path)
+            print("Contents:")
+            for item in os.listdir(zstacklib_path):
+                print("  - " + item)
+'''
+    
+    # Write and execute the test
+    test_file = "/tmp/debug_import_test.py"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/debug_import_test.py"

(S108)